docs/en/verifying-signatures.wml

```diff
@@ -210,113 +391,250 @@ Primary key fingerprint: EF6E 286D DA85 EA2A 4BA7  DE68 4E2C 6E87 9329 8290
-      <li>Verify the sha256sums-unsigned-build.txt file by executing this
-      command:
-      <pre>gpg --verify sha256sums-unsigned-build.txt.asc sha256sums-unsigned-build.txt</pre></li>
-      <li>You should see a message like "Good signature from &lt;DEVELOPER
-      NAME&gt;". If you don't, there is a problem. Try these steps again.</li>
-      <li>If you want to verify a Windows Tor Browser package you need to first
-      strip off the authenticode signature of it. Tools that can be used for
-      this purpose are <a href="http://osslsigncode.sourceforge.net">osslsigncode</a> and
-      <a href="http://forum.xda-developers.com/showthread.php?t=416175">delcert.exe</a>.
-      Assuming you have built e.g. <tt>osslsigncode</tt> on a Linux computer you can enter
-      <pre>/path/to/your/osslsigncode remove-signature &#92;
-        /path/to/your/&lt;TOR BROWSER FILE NAME&gt;.exe &lt;TOR BROWSER FILE NAME&gt;.exe</pre></li>
-      <li>Now you can take the sha256sum of the Tor Browser package. On
-      Windows you can use the <a href="http://md5deep.sourceforge.net/">
-      hashdeep utility</a> and run
-      <pre>C:\location\where\you\saved\hashdeep -c sha256sum &lt;TOR BROWSER FILE NAME&gt;.exe</pre>
-      <p>On Linux you can run</p>
-      <pre>sha256sum &lt;TOR BROWSER FILE NAME&gt;.tar.gz</pre>
-      without having to download a utility. Note: this does not work for OS X
-      yet due to Apple's codesigning requirement.</li>
-      <li>You will see a string of letters and numbers.</li>
-      <li>Open <tt>sha256sums-unsigned-build.txt</tt> in a text editor.</li>
-      <li>Locate the name of the Tor Browser file you downloaded.</li>
-      <li>Compare the string of letters and numbers to the left of your
-      filename with the string of letters and numbers that appeared
-      on your command line. If they match, you've successfully verified the
-      build.</li>
+      for Tor Browser <version-torbrowserbundlelinux64>.
+      </li>
+      <li>
+      Retrieve the signers' GPG key with following ID with the method <a href="#ImportKey">described above</a>:<br/>
+      (Other developers' key IDs can be found
+      <a href="<page docs/signing-keys>">here)</a>
+      </li>
+     </ul>
+     <pre>
+     0x4E2C6E8793298290
+     </pre>
+  </article>
+  <article class="ac-box">
+  
+   <p><span class="hint">Choose your operating system:</span>
+   <span>
+
+    <label for="ac-4-1"><a name="ChecksumsWindows">&#9658; Windows</a></label>
+   </span>
+   <!--<span>
+   <input id="ac-4-2" name="accordion-4" type="radio" />
+   <label for="ac-4-2"><a name="ChecksumMacOS">MacOS</a></label>
+   </span>-->
+   <span>
+    <label for="ac-4-3"><a name="ChecksumsLinux">&#9658; Linux</a></label>
+   </span>
+   </p>
+
+<!-- Windows -->
+  <div>
+    <input id="ac-4-1" name="accordion-4" type="radio" checked />
+   <article class="ac-os">
+    <!--<pre id="ttb-key">
+    > gpg.exe --keyserver pool.sks-keyservers.net --recv-keys 0x4E2C6E8793298290
```

Comment:
This is commented because I assume at this point it is ok, to tell users reproduce above step to import the key by just telling the key id. I see no better option at the moment.

Reply:
You should feel free to delete obsolete instructions.
We can always get them back out of git.